docs/getting_started/about_composer.md

```diff
@@ -160,27 +157,22 @@ Best practice for Bundles is described in Symfony documentation under [Best Pra
 #### Git repository naming
 
 -   You may omit vendor name in repository naming, assuming vendor name is reflected in organization / user account it is attached to.
--   You may also choose to follow composer package naming on repository name which is more relevant when trying to find a given package later.
+-   You may also choose to follow Composer package naming on repository name which is more relevant when trying to find a given package later.
 
 #### Composer Metadata
 
--   For defining `"type"`, the following are at the moment known valid values:
+-   For defining `type`, the following are at the moment known valid values:
     -   `ezplatform-bundle` | Symfony bundles that uses eZ Platform features
-    -   `ezstudio-bundle` | Symfony bundles that uses eZ Platform Enterprise Edition features*
-        *
+    -   `ezstudio-bundle` | Symfony bundles that uses eZ Platform Enterprise Edition features
         -   Deprecated: Please use ezplatform-bundle and add dependencies on the ee packages you depend on instead.
     -   `symfony-bundle` | Standard symfony bundles as described in Symfony doc.
 -   For eZ Publish (legacy) and eZ Publish Platform there where also:
     -   `ezpublish-legacy-extension` | For standalone 4.x (legacy) extensions, to be used with [ezpublish-legacy-installer](https://github.com/ezsystems/ezpublish-legacy-installer)
```

Comment:
Could we get rid of this line (and the whole "Publish" section) as well? As it's 4.x...

Reply:
According to Andre it should stay because of legacy bridge. But we can clean it up for master, so it will be done in a separate PR.